javascript/src/test/pages/Home/Home.test.js

```diff
@@ -19,4 +19,24 @@ describe("Home tests", () => {
   test("renders without crashing", () => {
     render(<Home/>);
   });
+
+  test("Test for checkbox Cancelled,Closed,Full", () => {
+    const { getByTestId } =  render(<Home/>);
+
+    const checkbox1 = getByTestId(`inline-checkbox-1`);
+    expect(checkbox1.checked).toEqual(false);
+    fireEvent.click(checkbox1);
+    expect(checkbox1.checked).toEqual(true);
+
+    const checkbox2 = getByTestId(`inline-checkbox-2`);
+    expect(checkbox2.checked).toEqual(false);
+    fireEvent.click(checkbox2);
+    expect(checkbox2.checked).toEqual(true);
+
+    const checkbox3 = getByTestId(`inline-checkbox-3`);
+    expect(checkbox3.checked).toEqual(false);
+    fireEvent.click(checkbox3);
+    expect(checkbox3.checked).toEqual(true);
+  });
+
```

Comment:
As per code review on the other PR, instead of:

* `inline-checkbox-1`,
* `inline-checkbox-2`, 
* `inline-checkbox-3`,

Use meaningful names:

* `inline-checkbox-closed`,
* `inline-checkbox-full`, 
* `inline-checkbox-cancelled`,

Reply:
This has now been fixed by the team